src/RecurringGrantDropWithRegistrar.sol

```diff
@@ -0,0 +1,193 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.19;
+
+import { IGrant } from './IGrant.sol';
+import { GrantRegistrar } from './GrantRegistrar.sol';
+import { Ownable } from 'openzeppelin-contracts/contracts/access/Ownable.sol';
+import { ERC20 } from 'openzeppelin-contracts/contracts/token/ERC20/ERC20.sol';
+import { ECDSA } from 'openzeppelin-contracts/contracts/utils/cryptography/ECDSA.sol';
+import { Ownable2Step } from 'openzeppelin-contracts/contracts/access/Ownable2Step.sol';
+import { SafeERC20 } from 'openzeppelin-contracts/contracts/token/ERC20/utils/SafeERC20.sol';
+
+/// @title RecurringGrantDrop
+/// @author Worldcoin
+contract RecurringGrantDrop is Ownable2Step {
```

Comment:
let's rename to `RecurringGrantDropWithRegistrar`

Reply:
I think it would be useful to make this contract pausable. Currently we have to do this by removing the allowance. I think good to have an additional option.